app/views/curation_concerns/file_sets/_show_actions.html.erb

```diff
@@ -1,5 +1,5 @@
   <div class="form-actions">
-    <%= link_to "Download this File", main_app.download_path(presenter), class: 'btn btn-default' %>
+    <%= link_to "Download this File", main_app.download_path(presenter), class: 'btn btn-default', target: :_blank %>
```

Comment:
I should say, I'm completely open to another fix and removing the `target` attribute from all of them. This fix is consistent and more convenient.

If this gets merged and #1050 closed, we could consider opening another issue to sort out the xhr issue.